charts/newrelic-prometheus/README.md

```diff
@@ -91,19 +91,40 @@ All pod and endpoints with the `prometheus.io/scrape: true` annotation will be s
 
 Check the [docs](https://github.com/newrelic/newrelic-prometheus-configurator/blob/main/docs/KubernetesTargetFilter.md) for more information about it usage.
 
+### Kubernetes job examples
+
+The following job allows to scrape API Server metrics:
```

Comment:
Should we mention that some info could be duplicated with what scrapes nri-kubernetes.

I would personally feel safer with an hardcoded service rather than a target_discovery. 
Any user in the cluster could create a service label it `component: apiserver` and steal our token.

Reply:
agree, good point! I will address both concerns